src/traffic_server/InkAPI.cc

```diff
@@ -773,7 +773,7 @@ isWriteable(TSMBuffer bufp)
 static MIMEFieldSDKHandle *
 sdk_alloc_field_handle(TSMBuffer /* bufp ATS_UNUSED */, MIMEHdrImpl *mh)
 {
-  MIMEFieldSDKHandle *handle = mHandleAllocator.alloc();
+  MIMEFieldSDKHandle *handle = THREAD_ALLOC(mHandleAllocator, this_thread());
```

Comment:
Do we need to get rid of `ClassAllocator<MIMEFieldSDKHandle> mHandleAllocator`?
https://github.com/apache/trafficserver/blob/c4b7eaac6ec3e98702b205895f914f2dcec76369/src/traffic_server/InkAPI.cc#L395

Reply:
NVM, they need to be the same name.

https://github.com/apache/trafficserver/blob/c4b7eaac6ec3e98702b205895f914f2dcec76369/iocore/eventsystem/I_ProxyAllocator.h#L74-L75